### ISSUE TITLE
Spanish Translation

### ISSUE THREAD

**Issue by dbucci24** [xxhomey19/nba-go#67]
Hello! I would like to translate the project into Spanish! is it already translated?

**Comment by xxhomey19** [comment]
Hi @dbucci24 
Actually, the data isn't translated to any language except English. It'll be awesome if it's translated to more different languages!

**Comment by dbucci24** [comment]
Yes I can do it. but I need you to be active to check my pull requests and give merge, as much as possible every day. Could be?

**Comment by xxhomey19** [comment]
I'm afraid I couldn't review PR everyday, but I promise I will do it as soon as possible.
